### ISSUE TITLE
[Feature request] Chrome Debugging

### ISSUE THREAD

**Issue by jefvel** [Kode/KodeStudio#26]
Is it possible to add a launch option to Kode Studio that simply runs the program in a browser, instead of an electron window?

**Comment by RobDangerous** [comment]
Yes, doing that for Chrome in particular is possible (other browsers would be more difficult).

**Comment by jefvel** [comment]
Would it be a quick fix? It'd be nice

**Comment by RobDangerous** [comment]
Yes, can probably add that in the next release.

**Comment by jefvel** [comment]
Nice, thanks!

**Comment by FaB0SS** [comment]
Any news?

**Comment by RobDangerous** [comment]
No but you can always just add launch configs yourself.
